cli/lib/nf3.mjs

```diff
@@ -1058,39 +1060,76 @@ class Nf3 {
     };
 
     connection.onmessage = async message => {
-      const msg = JSON.parse(message.data);
-      const { type, txDataToSign, block, transactions, data } = msg;
+      if (connection._ws && connection._ws.readyState === WebSocket.OPEN) {
```

Comment:
This was added because there is a test that constantly reboots the node. This additional code helps to support this scenario. In any case, tjñjis code is going away